Changelog.md

```diff
@@ -27,6 +27,7 @@ Bugfixes:
  * SMTChecker: Fix pointer knowledge erasing in loops.
  * SMTChecker: Fix internal error when using compound bitwise assignment operators inside branches.
  * SMTChecker: Fix internal error when inlining a function that returns a tuple containing an unsupported type inside a branch.
+ * SMTChecker: Fix internal error when inlining functions from a different source that use state variables.
```

Comment:
```suggestion
 * SMTChecker: Fix internal error when inlining functions from a different source that uses state variables.
```

Reply:
I think this is the same that Marenz suggested here: https://github.com/ethereum/solidity/pull/7170#discussion_r311650429 ;)
The `functions` use the state variables, not the `source`

Reply:
Maybe I should rephrase it, seems to be confusing